### PR TITLE
Add funding confirmation model

### PR DESCRIPTION
**Pull request by DH001**

### CODE REVIEW

File: src/main/java/uk/org/openbanking/datamodel/fund/OBCashAccountDebtor3.java [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+import io.swagger.annotations.ApiModelProperty;
+
+import java.util.Objects;
+
+/**
+ * Unambiguous identification of the account of the debtor to which a confirmation of funds consent will be applied.
+ */
+@ApiModel(description = "Unambiguous identification of the account of the debtor to which a confirmation of funds consent will be applied.")
+@javax.annotation.Generated(value = "io.swagger.codegen.languages.JavaClientCodegen", date = "2018-11-15T11:58:26.525Z")
+public class OBCashAccountDebtor3 {
```
Comment (benjefferies) [inline src/main/java/uk/org/openbanking/datamodel/fund/OBCashAccountDebtor3.java]: I think we refactored this as https://github.com/ForgeRock/OpenBanking-Java-SDK/blob/master/src/main/java/uk/org/openbanking/datamodel/account/OBCashAccount3.java

File: src/main/java/uk/org/openbanking/datamodel/fund/OBError1.java [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+
+import java.util.Objects;
+
+/**
+ * OBError1
+ */
+@javax.annotation.Generated(value = "io.swagger.codegen.languages.JavaClientCodegen", date = "2018-11-15T11:58:26.525Z")
+public class OBError1 {
```
Comment (benjefferies) [inline src/main/java/uk/org/openbanking/datamodel/fund/OBError1.java]: Think these exist as https://github.com/ForgeRock/OpenBanking-Java-SDK/blob/master/src/main/java/uk/org/openbanking/datamodel/error/OBError1.java

File: src/main/java/uk/org/openbanking/datamodel/fund/OBErrorResponse1.java [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * An array of detail error codes, and messages, and URLs to documentation to help remediation.
+ */
+@ApiModel(description = "An array of detail error codes, and messages, and URLs to documentation to help remediation.")
+@javax.annotation.Generated(value = "io.swagger.codegen.languages.JavaClientCodegen", date = "2018-11-15T11:58:26.525Z")
+public class OBErrorResponse1 {
```
Comment (benjefferies) [inline src/main/java/uk/org/openbanking/datamodel/fund/OBErrorResponse1.java]: Think it exists as https://github.com/ForgeRock/OpenBanking-Java-SDK/blob/master/src/main/java/uk/org/openbanking/datamodel/error/OBErrorResponse1.java

File: src/main/java/uk/org/openbanking/datamodel/fund/OBExternalRequestStatus1Code.java [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+package uk.org.openbanking.datamodel.fund;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * Specifies the status of consent resource in code form.
+ */
+public enum OBExternalRequestStatus1Code {
```
Comment (benjefferies) [inline src/main/java/uk/org/openbanking/datamodel/fund/OBExternalRequestStatus1Code.java]: Think there is an existing class we can use here

Comment (DH001) [reply src/main/java/uk/org/openbanking/datamodel/fund/OBExternalRequestStatus1Code.java]: Removed duplicate classes

**Pull request merged**